### PR TITLE
Fix `zombienet-bridges-0001-asset-transfer-works`

### PR DESCRIPTION
**Pull request by serban300**
Closes https://github.com/paritytech/polkadot-sdk/issues/6161

Westend BridgeHub freezes for a while at block 3 and if we try to init the bridge and fund the accounts during that time, it fails. So we wait untill all the parachains produced at least 10 blocks, in order to make sure that they work reliably.

### CODE REVIEW

File: .gitlab/pipeline/zombienet/bridges.yml [hunk 1]
```diff
@@ -47,8 +47,6 @@
     - cp -r /tmp/bridges-tests-run-*/bridge_hub_rococo_local_network/*.log ./zombienet-logs/
     # copy logs of westend nodes
     - cp -r /tmp/bridges-tests-run-*/bridge_hub_westend_local_network/*.log ./zombienet-logs/
-  tags:
```
Comment (pepoviola) [inline .gitlab/pipeline/zombienet/bridges.yml:50]: Ah, sorry about the noice. I set the tag for debugging in the zombienet cluster.

Comment (serban300) [reply .gitlab/pipeline/zombienet/bridges.yml:50]: Oh, ok. I see that without the tag it advances a bit further. :D Will remove it, but still have to investigate one more issue

**Comment by serban300** [comment]
> just out of curiosity:
> 
> > Westend BridgeHub freezes for a while at block 3
> 
> This is something new or it's been happening before and we just didn't realize that? Does it happen also for Rococo BridgeHub?

It's something new. I think it started happening after https://github.com/paritytech/polkadot-sdk/pull/5461 . And it only happens on Westend Bridge Hub. It doesn't happen on Rococo.

**Review by pepoviola — APPROVED** [review]

**Comment by bkontur** [comment]
> > just out of curiosity:
> > > Westend BridgeHub freezes for a while at block 3
> > 
> > 
> > This is something new or it's been happening before and we just didn't realize that? Does it happen also for Rococo BridgeHub?
> 
> It's something new. I think it started happening after #5461 . And it only happens on Westend Bridge Hub. It doesn't happen on Rococo.

cc: @alindima any idea?

**Pull request merged**